tests/serde.rs

```diff
@@ -180,11 +181,33 @@ fn numbers() {
         to_value(&std::u64::MAX).unwrap_err();
     }
 
-    // Test u64 and i64 bigint serialization feature
-    let big_int_serializer = Serializer::new().serialize_64_bit_numbers_as_big_int(true);
+    // By default serializing i128 and u128 results in an error
+    {
+        to_value(&0_i128).unwrap_err();
+        to_value(&0_u128).unwrap_err();
+    }
+
+    // By default deserializing i128 and u128 uses 64 bit implementation
+    {
+        assert_eq!(from_value::<i128>(JsValue::from(0_i128)).unwrap(), 0);
+        assert_eq!(from_value::<i128>(JsValue::from(42_i128)).unwrap(), 42);
+        assert_eq!(from_value::<i128>(JsValue::from(-42_i128)).unwrap(), -42);
+        assert_eq!(from_value::<u128>(JsValue::from(0_u128)).unwrap(), 0);
+        assert_eq!(from_value::<u128>(JsValue::from(42_u128)).unwrap(), 42);
+    }
+
+    // Test large number bigint serialization feature
+    let big_int_serializer = Serializer::new().serialize_large_number_types_as_big_ints(true);
     {
         const MAX_SAFE_INTEGER: i64 = 9_007_199_254_740_991;
 
+        // Should be bigint
+        0_i64
```

Comment:
Nit: instead of converting, you can probably just do `assert!(expr.is_bigint())`. Or does this one provide a better error message or something?